app/scrapers/ask.py

```diff
@@ -9,7 +9,7 @@ def __init__(cls):
         pass
 
     @classmethod
-    def get_page(cls,query):
+    def get_ask_page(cls,query):
```

Comment:
Changes should be relevant to the issue you are solving, this should be in another PR

Reply:
@AnshulMalik This error all of sudden was happening and making travis CI test fail that's why I edited it.

Reply:
Yep, I think https://github.com/fossasia/query-server/pull/205 will resolve that issue

Reply:
Ok, nice Will wait for that merge

Reply:
My sense is that #205 is no longer blocking progress on this PR.